### DISCUSSION TITLE
Render highcharts graphic inside a pdf file

### DISCUSSION THREAD

**Discussion by acgtwentyone** [dompdf/dompdf#2968] [General]
Hi, i wonder if there's a way to render a highcharts graphic inside a pdf file.

**Comment by bsweeney** [comment]
Dompdf itself can not render the charts since the charts are JS-based. You would have to render the chart independently then reference the image in the HTML. Maybe the [HighCharts export server](https://www.highcharts.com/docs/export-module/render-charts-serverside) would help?

**Reply by acgtwentyone** [reply]
@bsweeney thank you.
